docs/Configuration/yaml-files.md

```diff
@@ -246,10 +246,10 @@ controls:
     custom_settings:
       - path: ../lib/macos-profile1.mobileconfig
         labels_exclude_any:
-          - Label 1
+          - "Macs on Sequoia"
```

Comment:
@noahtalerman do these need quotes?
```suggestion
          - Macs on Sequoia
```

Reply:
It looks like no: https://www.w3schools.io/file/yaml-arrays/

I removed the quotes to be consistent w/ other arrays in the YAML reference.